restructuredtext_lint/cli.py

```diff
@@ -24,37 +27,51 @@ def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
             if format == 'text':
                 stream.write('INFO File {filepath} is clean.\n'.format(filepath=filepath))
         else:
-            error_occurred = True
-            if format == 'text':
-                for err in file_errors:
-                    # e.g. WARNING readme.rst:12 Title underline too short.
-                    stream.write('{err.type} {err.source}:{err.line} {err.message}\n'.format(err=err))
-            elif format == 'json':
-                error_dicts.extend({
-                    'line': error.line,
-                    'source': error.source,
-                    'level': error.level,
-                    'type': error.type,
-                    'message': error.message,
-                    'full_message': error.full_message,
-                } for error in file_errors)
+            file_error_level = max(error.level for error in file_errors)
+            error_level = max(file_error_level, error_level)
+            if file_error_level < report:
+                if format == 'text':
+                    stream.write('INFO File {filepath} has no errors above threshold {report}.\n'.format(
+                            filepath=filepath, report=report))
+            else:
+                if format == 'text':
+                    for error in file_errors:
+                        if error.level >= report:
+                            # e.g. WARNING readme.rst:12 Title underline too short.
+                            stream.write('{err.type} {err.source}:{err.line} {err.message}\n'.format(err=error))
+                elif format == 'json':
+                    error_dicts.extend({
+                        'line': error.line,
+                        'source': error.source,
+                        'level': error.level,
+                        'type': error.type,
+                        'message': error.message,
+                        'full_message': error.full_message,
+                    } for error in file_errors if int(error.level) >= report)
 
     if format == 'json':
         stream.write(json.dumps(error_dicts))
 
-    if error_occurred:
-        sys.exit(1)
+    if error_level >= report:
+        sys.exit(error_level)
     else:
+        # e.g. Run with report level 2, but only found level 1 (info)
         sys.exit(0)
 
 
 def main():
     # Set up options and parse arguments
-    parser = argparse.ArgumentParser(description='Lint reStructuredText files')
+    parser = argparse.ArgumentParser(description="""Lint reStructuredText files.
+
+                                     Returns zero if all files are clean, otherwise returns error
```

Comment:
Please use the numeric form of zero/`0` for consistency with other numeric forms